tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -166,15 +166,37 @@ export function getFilteredParent(apiItem: ApiItem): ApiItem | undefined {
 }
 
 /**
- * Adjusts the name of the item as needed.
- * Accounts for method overloads by adding a suffix such as "myMethod_2".
+ * Converts invalid filename characters to underscores.
+ */
+function getFileSafeNameForName(apiItemName: string): string {
```

Comment:
Code that was previously using this helper directly were (incorrectly) not getting the qualified naming for cases like overloads.